datasets/ResidentialACsAndHPsPerfCurves.idf

```diff
@@ -36,21 +36,23 @@
 !               Beginning of Single Speed Air Conditioners Performance Curves
 ! *******************************************************************************************
 
-  Coil:Cooling:DX:SingleSpeed,
-   ACDXCoil Name,           !- Name
-   ,                        !- Availability Schedule Name
-   ,                        !- Rated Total Cooling Capacity {W}
-   ,                        !- Rated Sensible Heat Ratio
-   ,                        !- Rated COP {W/W}
-   ,                        !- Rated Air Flow Rate {m3/s}
-   ,                        !- Rated Evaporator Fan Power Per Volume Flow Rate {W/(m3/s)}
-   ,                        !- Air Inlet Node Name
-   ,                        !- Air Outlet Node Name
-   ACCoolingCAPFTemp,       !- Total Cooling Capacity Function of Temperature Curve Name
-   ACCoolingCAPFFF,         !- Total Cooling Capacity Function of Flow Fraction Curve Name
-   ACCoolingEIRFTemp,       !- Energy Input Ratio Function of Temperature Curve Name
-   ACCoolingEIRFFF,         !- Energy Input Ratio Function of Flow Fraction Curve Name
-   ACCoolingPLFFPLR;        !- Part Load Fraction Correlation Curve Name
+! The coil objects in this idf are entered generically with many required fields purposely removed.  These objects are left in the file as comments to show how to use the curves defined in this file, which remain uncommented to allow including the curves in a particular idf, by macro, for example.
```

Comment:
@Myoldmopar Looks like this one needs this comment?
! The objects and syntax in this dataset comply with EnergyPlus, Version 8.4.0 and above, unless otherwise noted.